### PR TITLE
use an interval with data from 2 weeks ago so that incremental load w…

### PR DESCRIPTION
**Pull request by Geheiner**
…orks

### CODE REVIEW

**Comment by Geheiner** [comment]
With the previously provided time interval, you already downloaded the most recent data which made the incremental mode demonstration meaningless. Therefore, I moved the default timestamps 2 weeks in the past

File: part3/src/main/scala/io/smartdatalake/workflow/dataobject/CustomWebserviceDataObject.scala [hunk 1]
```diff
@@ -24,8 +24,8 @@ import scala.util.{Failure, Success}
 
 case class HttpTimeoutConfig(connectionTimeoutMs: Int, readTimeoutMs: Int)
 
-// Default to the interval of 2 days from now to today
-case class DepartureQueryParameters(airport: String, begin: Long = System.currentTimeMillis() / 1000 - 172800L, end: Long = System.currentTimeMillis() / 1000)
+// Default to the interval of [2 weeks and 2 days ago] -> [2 weeks ago]
+case class DepartureQueryParameters(airport: String, begin: Long = System.currentTimeMillis()/1000 - 1209600 - 172800L, end: Long = System.currentTimeMillis()/1000 - 1209600 )
```
Comment (ThierryGirod) [inline part3/src/main/scala/io/smartdatalake/workflow/dataobject/CustomWebserviceDataObject.scala:28]: Are there any code formatting rules to follow?

```suggestion
case class DepartureQueryParameters(airport: String, begin: Long = System.currentTimeMillis() / 1000 - 1209600 - 172800L, end: Long = System.currentTimeMillis() / 1000 - 1209600 )
```

File: part3/src/main/scala/io/smartdatalake/workflow/dataobject/CustomWebserviceDataObject.scala.solution [hunk 2]
```diff
@@ -24,8 +24,8 @@ import scala.util.{Failure, Success}
 
 case class HttpTimeoutConfig(connectionTimeoutMs: Int, readTimeoutMs: Int)
 
-// Default to the interval of 2 days from now to today
-case class DepartureQueryParameters(airport: String, begin: Long = System.currentTimeMillis() / 1000 - 172800L, end: Long = System.currentTimeMillis() / 1000)
+// Default to the interval of [2 weeks and 2 days ago] -> [2 weeks ago]
+case class DepartureQueryParameters(airport: String, begin: Long = System.currentTimeMillis()/1000 - 1209600 - 172800L, end: Long = System.currentTimeMillis()/1000 - 1209600 )
```
Comment (ThierryGirod) [inline part3/src/main/scala/io/smartdatalake/workflow/dataobject/CustomWebserviceDataObject.scala.solution:28]: Same here with the whitespaces.

```suggestion
case class DepartureQueryParameters(airport: String, begin: Long = System.currentTimeMillis() / 1000 - 1209600 - 172800L, end: Long = System.currentTimeMillis() / 1000 - 1209600 )
```

**Review by ThierryGirod — APPROVED** [review]
LGTM. You can accept the suggestions or leave them.

**Comment by Geheiner** [comment]
> LGTM. You can accept the suggestions or leave them.

In principle, we use IntelliJ default formatting rules, but don t enforce to apply them, this time I forgot it

**Pull request merged**